src/cargo.rs

```diff
@@ -132,12 +132,27 @@ impl CommandCargoExt for process::Command {
 pub(crate) fn cargo_bin_cmd<S: AsRef<str>>(name: S) -> Result<process::Command, CargoError> {
     let path = cargo_bin(name);
     if path.is_file() {
-        Ok(process::Command::new(path))
+        if let Some(runner) = cargo_runner() {
+            let runner_vec: Vec<_> = runner.split(" ").collect();
```

Comment:
What is the space-escaping definition for the target running?  I'm assuming we need something like `shlex`

Reply:
I haven't found the info in the cargo manifest documentation, but indeed, shlex is a better option :+1:

Reply:
I can't find how to make shlex produce any meaningful error message in case of parsing error, so I've just ignored the runner configuration for now if it's not parsable. I'll try to find how cargo is doing it and make it cleaner

Reply:
mmm, shlex doesn't seem to be the right option either: 

> /// This supports a list of strings like `['/path/to/program', 'somearg']`
> /// or a space separated string like `'/path/to/program somearg'`.

https://github.com/rust-lang/cargo/blob/master/src/cargo/util/config/path.rs#L46

Reply:
Was just looking at those docs so it seems that its much simpler of a model than I expected.

Reply:
I've tried several setting the runner to several formats like `['/path/to/program', 'somearg']` or `'/path/to/program somearg'` — only the version without any quote works.

So I'm going back to the simple white space split :-)